src/cli/shared/Scope.option.ts

```diff
@@ -0,0 +1,30 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright IBM Corp, 2019
+*
+*/
+
+import { ICommandOptionDefinition } from "@brightside/imperative";
+
+const MAX_LENGTH = 8;
+
+/**
+ * Imperative option for the "scope" parameter
+ *
+ */
+export const ScopeOption: ICommandOptionDefinition = {
+    name: "scope",
+    aliases: ["sc"],
+    type: "string",
+    stringLengthRange: [1, MAX_LENGTH],
+    impliesOneOf: ["csdgroup", "resgroup"],
+    description: "Specifies the name of the CICS System, or CICS System Group " +
```

Comment:
```suggestion
    description: "Specifies the name of the CICS system, or CICS system group " +
```

Reply:
I'm happy to change it, but the casing is as used in the DFHDPLOY doc in the KC.